addons/account/views/account_bank_statement_views.xml

```diff
@@ -67,6 +67,28 @@
               </p>
             </field>
         </record>
+
+        <record id="action_credit_statement_tree" model="ir.actions.act_window">
+            <field name="name">Credit Statements</field>
+            <field name="res_model">account.bank.statement</field>
+            <field name="view_mode">tree,pivot,graph</field>
+            <field name="domain">['|', ('journal_id', '=', False), ('journal_id.type', '=', 'credit')]</field>
```

Comment:
`('journal_id', '=', False)`?

Reply:
We do it similarly for Bank statement

Reply:
Why?

Reply:
Do I remove it for both ? :man_shrugging:

Reply:
Sure